src/cli/set-env-var.test.ts

```
@@ -32,6 +33,12 @@ await tmp.withDir(async (dir) => {
   process.env.KIT_CONTEXT = "workflow"
   process.env.KENV = join(dir.path, ".kenv")
 
+  // Hard exit if KENV = ~/.kenv to avoid deleting the user's .env
+  if (path.resolve(process.env.KENV) === path.resolve(os.homedir(), ".kenv")) {
+    throw new Error("KENV is set to ~/.kenv, which is not allowed in tests")
+    process.exit(1)
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)